From 3bdb5e20b14385f089ba20d77baf30085b2cca9e Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <bochecha@daitauha.fr>
Date: Mon, 9 Oct 2017 02:26:59 +0200
Subject: [PATCH 1/3] Rename the icons, appdata and desktop files

This is good practice, and it helps a lot with Flatpak.

https://bugzilla.gnome.org/show_bug.cgi?id=788693
---
 configure.ac                                          |   2 +-
 data/Makefile.am                                      |   4 ++--
 .../apps/{d-feet.svg => org.gnome.d-feet.svg}         |   0
 data/icons/Makefile.am                                |  18 +++++++++---------
 .../16x16/apps/{d-feet.png => org.gnome.d-feet.png}   | Bin
 .../24x24/apps/{d-feet.png => org.gnome.d-feet.png}   | Bin
 .../256x256/apps/{d-feet.png => org.gnome.d-feet.png} | Bin
 .../32x32/apps/{d-feet.png => org.gnome.d-feet.png}   | Bin
 .../48x48/apps/{d-feet.png => org.gnome.d-feet.png}   | Bin
 .../apps/{d-feet.svg => org.gnome.d-feet.svg}         |   0
 ...appdata.xml.in => org.gnome.d-feet.appdata.xml.in} |   2 +-
 ...t.desktop.in.in => org.gnome.d-feet.desktop.in.in} |   2 +-
 data/ui/bus.ui                                        |   2 +-
 po/POTFILES.in                                        |   4 ++--
 po/POTFILES.skip                                      |   2 +-
 15 files changed, 18 insertions(+), 18 deletions(-)
 rename data/icons/HighContrast/scalable/apps/{d-feet.svg => org.gnome.d-feet.svg} (100%)
 rename data/icons/hicolor/16x16/apps/{d-feet.png => org.gnome.d-feet.png} (100%)
 rename data/icons/hicolor/24x24/apps/{d-feet.png => org.gnome.d-feet.png} (100%)
 rename data/icons/hicolor/256x256/apps/{d-feet.png => org.gnome.d-feet.png} (100%)
 rename data/icons/hicolor/32x32/apps/{d-feet.png => org.gnome.d-feet.png} (100%)
 rename data/icons/hicolor/48x48/apps/{d-feet.png => org.gnome.d-feet.png} (100%)
 rename data/icons/hicolor/scalable/apps/{d-feet.svg => org.gnome.d-feet.svg} (100%)
 rename data/{d-feet.appdata.xml.in => org.gnome.d-feet.appdata.xml.in} (96%)
 rename data/{d-feet.desktop.in.in => org.gnome.d-feet.desktop.in.in} (91%)

diff --git a/configure.ac b/configure.ac
index bba301b..bdeee4c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -48,7 +48,7 @@ AC_CONFIG_FILES([
 	src/dfeet/Makefile
 	src/tests/Makefile
 	data/Makefile
-	data/d-feet.desktop.in
+	data/org.gnome.d-feet.desktop.in
 	data/icons/Makefile
 	data/ui/Makefile
 	help/Makefile
diff --git a/data/Makefile.am b/data/Makefile.am
index c47e922..98f3e23 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -6,7 +6,7 @@ gsettings_SCHEMAS = org.gnome.d-feet.gschema.xml
 @INTLTOOL_DESKTOP_RULE@
 
 desktopdir = $(datadir)/applications
-desktop_in_files = d-feet.desktop.in
+desktop_in_files = org.gnome.d-feet.desktop.in
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 
 UPDATE_DESKTOP = if test -z "$(DESTDIR)"; then update-desktop-database $(datadir)/applications || :; fi
@@ -14,7 +14,7 @@ UPDATE_DESKTOP = if test -z "$(DESTDIR)"; then update-desktop-database $(datadir
 @INTLTOOL_XML_RULE@
 appdatadir = $(datadir)/appdata
 appdata_DATA = $(appdata_in_files:.xml.in=.xml)
-appdata_in_files = d-feet.appdata.xml.in
+appdata_in_files = org.gnome.d-feet.appdata.xml.in
 
 
 install-data-hook:
diff --git a/data/icons/HighContrast/scalable/apps/d-feet.svg b/data/icons/HighContrast/scalable/apps/org.gnome.d-feet.svg
similarity index 100%
rename from data/icons/HighContrast/scalable/apps/d-feet.svg
rename to data/icons/HighContrast/scalable/apps/org.gnome.d-feet.svg
diff --git a/data/icons/Makefile.am b/data/icons/Makefile.am
index cb07dcc..4e7c21d 100644
--- a/data/icons/Makefile.am
+++ b/data/icons/Makefile.am
@@ -1,6 +1,6 @@
 icon16dir = $(datadir)/icons/hicolor/16x16/apps
 icon16_DATA = \
-	hicolor/16x16/apps/d-feet.png \
+	hicolor/16x16/apps/org.gnome.d-feet.png \
 	hicolor/16x16/apps/dfeet-method-category.png \
 	hicolor/16x16/apps/dfeet-method.png \
 	hicolor/16x16/apps/dfeet-object.png \
@@ -11,28 +11,28 @@ icon16_DATA = \
 	hicolor/16x16/apps/dfeet-icon-default-service.png
 
 icon24dir = $(datadir)/icons/hicolor/24x24/apps
-icon24_DATA = hicolor/24x24/apps/d-feet.png
+icon24_DATA = hicolor/24x24/apps/org.gnome.d-feet.png
 
 icon32dir = $(datadir)/icons/hicolor/32x32/apps
-icon32_DATA = hicolor/32x32/apps/d-feet.png
+icon32_DATA = hicolor/32x32/apps/org.gnome.d-feet.png
 
 icon48dir = $(datadir)/icons/hicolor/48x48/apps
-icon48_DATA = hicolor/48x48/apps/d-feet.png
+icon48_DATA = hicolor/48x48/apps/org.gnome.d-feet.png
 
 #icon64dir = $(datadir)/icons/hicolor/64x64/apps
-#icon64_DATA = hicolor/64x64/apps/d-feet.png
+#icon64_DATA = hicolor/64x64/apps/org.gnome.d-feet.png
 
 #icon128dir = $(datadir)/icons/hicolor/128x128/apps
-#icon128_DATA = hicolor/128x128/apps/d-feet.png
+#icon128_DATA = hicolor/128x128/apps/org.gnome.d-feet.png
 
 icon256dir = $(datadir)/icons/hicolor/256x256/apps
-icon256_DATA = hicolor/256x256/apps/d-feet.png
+icon256_DATA = hicolor/256x256/apps/org.gnome.d-feet.png
 
 scalabledir = $(datadir)/icons/hicolor/scalable/apps
-scalable_DATA = hicolor/scalable/apps/d-feet.svg
+scalable_DATA = hicolor/scalable/apps/org.gnome.d-feet.svg
 
 scalablehcdir = $(datadir)/icons/HighContrast/scalable/apps
-scalablehc_DATA = HighContrast/scalable/apps/d-feet.svg
+scalablehc_DATA = HighContrast/scalable/apps/org.gnome.d-feet.svg
 
 
 gtk_update_icon_cache = gtk-update-icon-cache -f -t $(datadir)/icons/hicolor
diff --git a/data/icons/hicolor/16x16/apps/d-feet.png b/data/icons/hicolor/16x16/apps/org.gnome.d-feet.png
similarity index 100%
rename from data/icons/hicolor/16x16/apps/d-feet.png
rename to data/icons/hicolor/16x16/apps/org.gnome.d-feet.png
diff --git a/data/icons/hicolor/24x24/apps/d-feet.png b/data/icons/hicolor/24x24/apps/org.gnome.d-feet.png
similarity index 100%
rename from data/icons/hicolor/24x24/apps/d-feet.png
rename to data/icons/hicolor/24x24/apps/org.gnome.d-feet.png
diff --git a/data/icons/hicolor/256x256/apps/d-feet.png b/data/icons/hicolor/256x256/apps/org.gnome.d-feet.png
similarity index 100%
rename from data/icons/hicolor/256x256/apps/d-feet.png
rename to data/icons/hicolor/256x256/apps/org.gnome.d-feet.png
diff --git a/data/icons/hicolor/32x32/apps/d-feet.png b/data/icons/hicolor/32x32/apps/org.gnome.d-feet.png
similarity index 100%
rename from data/icons/hicolor/32x32/apps/d-feet.png
rename to data/icons/hicolor/32x32/apps/org.gnome.d-feet.png
diff --git a/data/icons/hicolor/48x48/apps/d-feet.png b/data/icons/hicolor/48x48/apps/org.gnome.d-feet.png
similarity index 100%
rename from data/icons/hicolor/48x48/apps/d-feet.png
rename to data/icons/hicolor/48x48/apps/org.gnome.d-feet.png
diff --git a/data/icons/hicolor/scalable/apps/d-feet.svg b/data/icons/hicolor/scalable/apps/org.gnome.d-feet.svg
similarity index 100%
rename from data/icons/hicolor/scalable/apps/d-feet.svg
rename to data/icons/hicolor/scalable/apps/org.gnome.d-feet.svg
diff --git a/data/d-feet.appdata.xml.in b/data/org.gnome.d-feet.appdata.xml.in
similarity index 96%
rename from data/d-feet.appdata.xml.in
rename to data/org.gnome.d-feet.appdata.xml.in
index 6fb5689..87c4bb6 100644
--- a/data/d-feet.appdata.xml.in
+++ b/data/org.gnome.d-feet.appdata.xml.in
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2013 Thomas Bechtold <thomasbechtold@jpberlin.de> -->
 <component type="desktop">
-  <id>d-feet.desktop</id>
+  <id>org.gnome.d-feet.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <_name>D-Feet</_name>
diff --git a/data/d-feet.desktop.in.in b/data/org.gnome.d-feet.desktop.in.in
similarity index 91%
rename from data/d-feet.desktop.in.in
rename to data/org.gnome.d-feet.desktop.in.in
index 46f86c0..6581f0c 100644
--- a/data/d-feet.desktop.in.in
+++ b/data/org.gnome.d-feet.desktop.in.in
@@ -4,7 +4,7 @@ _GenericName=D-Bus Debugger
 _Comment=Debug D-Bus applications
 _Keywords=debug;d-bus;dbus;dfeet;
 Exec=@PACKAGE@
-Icon=@PACKAGE@
+Icon=org.gnome.d-feet
 Terminal=false
 Type=Application
 Categories=GNOME;GTK;Development;Debugger;
diff --git a/data/ui/bus.ui b/data/ui/bus.ui
index 67aed2a..6708243 100644
--- a/data/ui/bus.ui
+++ b/data/ui/bus.ui
@@ -54,7 +54,7 @@
         <property name="visible">True</property>
         <property name="can_focus">False</property>
         <property name="pixel_size">128</property>
-        <property name="icon_name">d-feet</property>
+        <property name="icon_name">org.gnome.d-feet</property>
       </object>
       <packing>
         <property name="expand">True</property>
diff --git a/po/POTFILES.in b/po/POTFILES.in
index ea29e57..e73de53 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -1,7 +1,7 @@
 # List of source files containing translatable strings.
 # Please keep this file sorted alphabetically.
-data/d-feet.appdata.xml.in
-data/d-feet.desktop.in.in
+data/org.gnome.d-feet.appdata.xml.in
+data/org.gnome.d-feet.desktop.in.in
 data/org.gnome.d-feet.gschema.xml
 [type: gettext/glade]data/ui/addconnectiondialog.ui
 [type: gettext/glade]data/ui/app-menu.ui
diff --git a/po/POTFILES.skip b/po/POTFILES.skip
index 138c665..45f9f4b 100644
--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -1 +1 @@
-data/d-feet.desktop.in
+data/org.gnome.d-feet.desktop.in
-- 
2.14.3

